_data/beps_completed.yml

```
@@ -13,13 +13,19 @@
     - name: Kirstie Whitaker
   content:
     - raw
+  google_doc_created:
```

```suggestion
  google_doc_created: 2017-02
```

https://groups.google.com/g/bids-discussion/c/wzIAwCYVTbo/m/SuVHpYqrBAAJ